### PR TITLE
Add PostgREST installation using package managers to tut0

### LINKED ISSUE #702: Link to Installation from tutorial
## Problem

Users are trying the binaries instead of their own package manager. See https://app.gitter.im/#/room/#begriffs_postgrest:gitter.im

## Solution

Link to https://postgrest.org/en/stable/explanations/install.html#installation

Ideally the installation commands would be inline. Do we have an option besides duplicating the installation section?

I saw this https://sphinx-preview.readthedocs.io/en/latest/ but maybe there are other options.

### PR DESCRIPTION
**Pull request by laurenceisla**
Closes #702

- Used [the shared file approach to avoid duplicating code](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/reuse.html#include-a-shared-file),
- Also Modified all references of "OS X" to "macOS" (ref: https://en.wikipedia.org/wiki/MacOS#cite_note-10)

### CODE REVIEW

File: docs/shared/installation.rst
[the remark this answers is not in the document]
Comment (laurenceisla) [reply docs/shared/installation.rst:1]: Ah, good question. Just tried to look for terms in the search bar and it finds it as <no-title>. Can be accessed directly https://postgrest--706.org.readthedocs.build/en/706/shared/installation.html

I'll look how to hide it from search/access.

File: docs/tutorials/tut0.rst [hunk 1]
```diff
@@ -38,7 +38,25 @@ This will run the Docker instance as a daemon and expose port 5433 to the host s
...
+.. code-block:: bash
+
+  postgrest -h
+
+It should print the help page with its version and the available options.
+
+Downloading a Pre-Built Binary
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+PostgREST is also distributed as a single binary, with versions compiled for major distributions of macOS, Windows, Linux and FreeBSD. Visit the `latest release <https://github.com/PostgREST/postgrest/releases/latest>`_ for a list of downloads. In the event that your platform is not among those already pre-built, see :ref:`build_source` for instructions how to build it yourself. Also let us know to add your platform in the next release.
```
[the remark this answers is not in the document]
Comment (laurenceisla) [reply docs/tutorials/tut0.rst:59]: Yeah, I was thinking the same thing when separating both installation options. I don't quite get the explanation on "Other Linux" and "Other OS"  though.

File: docs/shared/installation.rst
[the remark this answers is not in the document]
Comment (steve-chavez) [reply docs/shared/installation.rst:1]: No biggie if it shows like that. Should be fine to leave as is if it's too complicated.

[the remark this answers is not in the document]
Comment (laurenceisla) [reply docs/shared/installation.rst:1]: Cool, it was straightforward enough, now the link above shows a 404 instead.

File: docs/tutorials/tut0.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (laurenceisla) [reply docs/tutorials/tut0.rst:59]: > "Other OS": On this tab we can put the instructions for building from source. Could be too big as a tab though. Not sure.

Yeah, the [building from source section](https://postgrest.org/en/latest/explanations/install.html#building-from-source) is already big. But the tab can have a link to that section, I don't think that should be a problem.

[the remark this answers is not in the document]
Comment (laurenceisla) [reply docs/tutorials/tut0.rst:59]: I was trying to do it that way, but found it a bit complex, I opened the issue #707 to continue there.

**Pull request merged**